.github/ISSUE_TEMPLATE/00-bug-performance-issue.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug/Performance Issue
+about: Use this template for reporting a bug or a performance issue.
+
+---
+
+<em>Please make sure that this is a bug. We only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub.</em>
+
+**System information**
+- Have I written custom code (as opposed to using a stock example script provided in Swiftplot):
+- OS Platform and Distribution (e.g., Linux Ubuntu 18.04):
+- Swift for TensorFlow version:
```

Comment:
Swift for TensorFlow is separate from SwiftPlot, recommend changing just to "Swift version:"

Reply:
same for the other templates